test/runtests.jl

```diff
@@ -83,6 +83,16 @@ end
     @test C(-Inf..1, IntervalBox(0.5..1.5, 2)) == IntervalBox(0.5..0.5, 2)
 end
 
+@testset "Contractors for functions having parameters" begin
+
+    @parameters r
+    vars =@variables x
+    f(x) = r*x*(1-x)
```

Comment:
I guess this should be `f(r, x)`, i.e. be a function of the parameters too?

Reply:
We won't do it because I think in function definition we should only specify variables, not parameters.

Reply:
We have to put the parameters too, otherwise we will be accessing global variables inside the function, which will be slow.

Reply:
Try timing the two versions. I expect it should be much slower if the parameters are not arguments of the function.

Reply:
Parameter r() are behaving exactly like variables x() and y(), How can they access global variables ?

Reply:
What is the point of worrying about global variable ?